packages/editor/src/utils/editor.ts

```diff
@@ -81,7 +84,7 @@ export const createBlockHub: (
       }
 
       // In edgeless mode.
-      const pageBlock = getEdgelessPage(page);
+      const pageBlock = getEdgelessPage(page) || getDefaultPage(page);
```

Comment:
Thank you for your contribution, but in this position, we should only handle the logic of Edgeless. I will make adjustments to your pull request to ensure it fits with the existing logic.

Reply:
Yes, I think so. However, I have no idea what "Edgeless" refers to. So I can't distinguish when it is Edgeless.